pkg/kubelet/prober/prober_manager.go

```diff
@@ -31,6 +32,16 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/format"
 )
 
+// ProberResults stores the results of a probe as prometheus metrics.
+var ProberResults = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
```

Comment:
Should this set a namespace?

Reply:
I don't think any of the metrics declared in pkg/kubelet set the namespace